blog/_posts/2017-09-24-we-heart-summer-of-code.md

```diff
@@ -0,0 +1,79 @@
+---
+title: We ❤️ Rails Girls Summer of Code
+layout: post
+created_at: Sunday 24 September 2017
+permalink: blog/2017-09-24-we-heart-summer-of-code
+current: blog
+author: Emma Deacon
+twitter: emmamdeacon
+categories:
+- student-posts
+---
+
+*This post contains songs we recommend listening to whilst reading - we felt that this incredible summer needed an incredible soundtrack to tell its story*
+
+
+We have had an incredible summer: it has been insanely busy and at times overwhelming, but we have both grown enormously as developers and team members. Rails Girls Summer of Code was one of the best decisions we have ever made, and it’s been a pleasure to be part of this superb and inspirational community. Being part of Rails Girls Summer of Code is an incredible opportunity, and it has been a transformative summer for us.
+
+We started the summer enthusiastically, wanting to contribute substantively to Babel. We felt like we needed to take this opportunity to become passionate open source developers.
+
+Rails Girls Summer of Code taught us a lot about the importance of communication, and perseverance. We have received generous support from the Babel community; nonetheless it was disquieting to be joining an open source community where we couldn’t find any other female members.
+
+[![Madonna, What It Feels like for a Girl ](http://img.youtube.com/vi/VQChkSgPCJ4/0.jpg)](http://www.youtube.com/watch?v=VQChkSgPCJ4)
```

Comment:
I think it would make more sense to use the markdown style to embed youtube video instead of having image which redirects to a video. Would like to do that change at all places?

Reply:
I did try and find how to do this, but couldn't figure it out/find the answer by Googling. Do you know how to that?

Reply:
Do you mean as an HTML video? Sorry, my Markdown skills aren't as good as they could be 🙈

Reply:
Yes, you can use iframe for that. For example following works for me:

```
<iframe width="560" height="315" src="https://www.youtube.com/embed/MHi9mKq0slA" frameborder="0" allowfullscreen></iframe>
```

Reply:
Okay, thanks! I have done that in the most recent PR; in preview in Atom they work, in View on GitHub the video links do not seem to show?